modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators/ExtensionMethods.cs

```diff
@@ -48,6 +49,21 @@ public static bool InheritsFrom(this ITypeSymbol? symbol, string assemblyName, s
             return false;
         }
 
+        public static string? GetDocumentationSummaryText(this ISymbol symbol)
+        {
+            var syntax = symbol.DeclaringSyntaxReferences.FirstOrDefault()?.GetSyntax();
+            while (syntax is VariableDeclaratorSyntax vds)
+            {
+                syntax = vds.Parent?.Parent;
+            }
+            var trivia = syntax?.GetLeadingTrivia();
+
+            string summaryContent = Regex.Match(trivia.ToString(), "(?<=<summary>)[.\\s\\S]*?(?=</summary>)", RegexOptions.Singleline).Value;
+            string cleanedSummary = Regex.Replace(summaryContent, @"^\s*///\s?", "", RegexOptions.Multiline);
+
+            return cleanedSummary.Trim().Replace("\n", "\\n").Replace("\r", "\\r");
```

Comment:
Same as above, because `GetDocumentationCommentXml` could no longer be used, it was changed to the current approach. The trivia extracts a string, which needs to be converted to XML. Personally, I think using Regex directly would be simpler.